packages/eslint-plugin/src/rules/naming-convention.ts

```diff
@@ -631,12 +631,10 @@ export default createRule<Options, MessageIds>({
 
       // #region interface
 
-      'TSEnumMember[computed != true]': {
-        handler: (
-          node: TSESTree.TSEnumMemberNonComputedName,
-          validator,
-        ): void => {
-          const id = node.id;
+      TSEnumMember: {
+        handler: (node: TSESTree.TSEnumMember, validator): void => {
+          // Unknown reason, can't get the correct type
+          const id = node.id as TSESTree.Identifier | TSESTree.StringLiteral;
```

Comment:
I think the build script is using the published package instead of local one. I tried to use `workspace:^`, but still can't build.